.gitmux.yml

```diff
@@ -83,3 +83,4 @@ tmux:
         hide_clean: false
         # Swaps order of behind & ahead upstream counts - "↓·1↑·1" -> "↑·1↓·1"
         swap_divergence: false
+        divergence_space: false
```

Comment:
This misses a doc string. You can use the same one as the readme.
The content of this file is output for `gitmux -printfg`